src/index.ts

```diff
@@ -33,14 +33,16 @@ export const run = () => {
   }
 };
 
-const showHelp = (cmdPassed: string, helpFlag: boolean) => {
-  if (cmdPassed) {
-    console.log(colors.red(`unknown command: ${cmdPassed}`), '\n');
-  } else if (!helpFlag) {
+const showHelp = (args: string[], helpFlag: boolean) => {
+  if (args.length > 2) {
+    console.log(colors.red('Too many arguments passed.'), '\n');
+  } else if (args[0]) {
+    console.log(colors.red(`unknown command: ${args[0]}`), '\n');
+  } else if (!args[0] && !helpFlag) {
     console.log(colors.red('No command passed.'), '\n');
   }
 
-  console.log(`Usage: ${colors.cyan('npx @nightwatch/mobile-helper COMMAND [options]')}\n`);
+  console.log(`Usage: ${colors.cyan('npx @nightwatch/mobile-helper COMMAND [options] [subcommand] [subcommand-options]')}\n`);
```

Comment:
This is the main help (which comprises the usages of both `android` and `ios`), so we shouldn't put `[subcommand] [subcommand-options]` here as such a thing is not applicable for the iOS setup.

Instead, we can do: `npx @nightwatch/mobile-helper COMMAND [args|options]`